validation/jinja2/validation/list_phrases.html

```diff
@@ -21,70 +21,80 @@
-      <div class="card__header card__header--grey" data-phrase-id={{ phrase.id }}>
-        &nbsp;
-    </div>
-  {% endif %}
-    <div class="card__body">
-      <p class="card__title--cree">{{ phrase.transcription }}</p>
-      <p class="card__title--english">Translation: {{ phrase.translation }}</p>
-      <p>Is this translation accurate?
-          {% if phrase.status == "linked" %}
-              <button class="button button--success-solid translation-judgement-accuracy-yes" data-phrase-id={{ phrase.id }}>Yes</button>
-              <button class="button button--fail translation-judgement-accuracy-no" data-phrase-id={{ phrase.id }}>No</button></p>
-          {% else %}
-              <button class="button button--success translation-judgement-accuracy-yes" data-phrase-id={{ phrase.id }}>Yes</button>
-              <button class="button button--fail translation-judgement-accuracy-no" data-phrase-id={{ phrase.id }}>No</button></p>
-          {%  endif %}
-        <table class="card__body--table">
-          <th>Recording</th>
-          <th>Speaker</th>
-          <th>This recording is...</th>
+            <div class="card__body">
+                <p class="card__title--cree">{{ phrase.transcription }}</p>
+                <p class="card__title--english">Translation: {{ phrase.translation }}</p>
 
-          {% for recording in phrase.recordings %}
-              <tr>
-            <td>
-              <div class="rec-height">
-                {{ macros.player(recording) }}
-              </div>
-            </td>
-              <td>
-                  {{ macros.speaker(recording) }}
-              </td>
-              <td>
-              <button class="button button--success">Good</button> <button class="button button--fail">Bad</button>
-              </td>
-              </tr>
+                {% if is_community %}
+                    <p>Is this translation accurate?
 
-          {% endfor %}{# for recording in phrase.recordings #}
-      </table>
-    {% if is_linguist %}
-      <a href={{ url('validation:segment_detail', phrase.id) }} class="button button--success right">More Options</a>
-    {% endif %} {# is linguist #}
-    </div>
-</div>
-  {% endfor %} {# phrase in phrases #}
+                    {% if phrase.status == "linked" %}
+                        <button class="button button--success-solid translation-judgement-accuracy-yes" data-phrase-id={{ phrase.id }}>Yes</button>
+                        <button class="button button--fail translation-judgement-accuracy-no" data-phrase-id={{ phrase.id }}>No</button></p>
+                    {% else %}
+                        <button class="button button--success translation-judgement-accuracy-yes" data-phrase-id={{ phrase.id }}>Yes</button>
+                        <button class="button button--fail translation-judgement-accuracy-no" data-phrase-id={{ phrase.id }}>No</button></p>
+                    {%  endif %}
+                {% endif %} {# check for instructor privileges #}
```

Comment:
Wait—I think the matching `{% if ` tag for this `endif` is on line 53, `{% if is_community %}`

Reply:
Oh, hmm. I updated the comments, do they make more sense now?

Reply:
Yeah. Honestly, I’d prefer these end-tag comments not be there at all since it’s so easy for them to get out of sync. But if you find them helpful, that’s much more important than my preferences here.

I am surprised and saddened though that PyCharm doesn’t seem to have a shortcut key to jump from the `endif` or `endfor` to the `if` or `for`.